src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/misc/StrangeNetherGoo.java

```diff
@@ -47,15 +47,12 @@ public int getBarteringLootChance() {
     @Nonnull
     @Override
     public ItemUseHandler getItemHandler() {
-        return this::use;
-    }
-
-    public void use(PlayerRightClickEvent e) {
-        if (e.getClickedBlock().isPresent()) {
-            if (Tag.SIGNS.isTagged(e.getClickedBlock().get().getType())) {
+        return e -> {
+            Optional<Block> block = e.getClickedBlock();
+            if (block.isPresent() && Tag.SIGNS.isTagged(block.get().getType())) {
```

Comment:
An empty line before the if always looks a bit nicer
```suggestion
            Optional<Block> block = e.getClickedBlock();

            if (block.isPresent() && Tag.SIGNS.isTagged(block.get().getType())) {
```